wg-data/README.md

```diff
@@ -0,0 +1,33 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://github.com/kubeflow/community/blob/master/generator/README.md
+--->
+# Data Working Group
+
+The WG "Data" is focused on enhancing the support for data/metadata-related tasks within Kubeflow, with a specific focus on the Spark operato and Model Registry. The group aims to simplify and improve data processing between various stages of ML lifecycle. For example, from Data Preparation to model training and fine-tuning. The group also aims to facilitate the ML model's metadata management, while ensuring seamless integration with other Kubeflow components. The goal of Spark on Kubernetes Operator is to simplify the capability of running Apache Spark on Kubernetes. It automates deployment and simplifies lifecycle management of Spark Jobs on Kubernetes. The goal of Model Registry is gather, analyze, and develop model registry requirements of Kubeflow community users.
```

Comment:
Suggested addition:

```
The goal of Feast is to provide a customizable, operational data system that enables ML Platform Engineers, Machine Learning Engineers, and Data Scientists to accelerate production machine learning.
```

Reply:
done in the yaml with fa3c3187d1d731bfe723c9c9b633a868baad051c